components/Header.tsx

```diff
@@ -1,74 +1,130 @@
 // Copyright 2023 the Deno authors. All rights reserved. MIT license.
 import {
   ACTIVE_LINK_STYLES,
-  BUTTON_STYLES,
   LINK_STYLES,
-  NAV_STYLES,
   SITE_BAR_STYLES,
   SITE_NAME,
 } from "@/utils/constants.ts";
 import { stripe } from "@/utils/payments.ts";
-import { Bell, CircleFilled } from "./Icons.tsx";
-import { getToggledStyles } from "@/utils/display.ts";
+import { Bars, Bell, CircleFilled, Cross } from "./Icons.tsx";
+import { cx } from "@twind/core";
+import type { JSX } from "preact";
+
+interface MenuItemProps {
+  name: string | JSX.Element;
+  href: string;
+  active?: boolean;
+  class?: string;
+  ariaLabel?: string;
+}
+
+function MenuItem(props: MenuItemProps) {
+  return (
+    <a
+      href={props.href}
+      class={cx(
+        props.active ? ACTIVE_LINK_STYLES : LINK_STYLES,
+        "relative text-gray-500 px-3 py-4 sm:py-2",
```

Comment:
```suggestion
        "relative text-gray-500 py-4 sm:py-2",
```
Perhaps, we don't need any horizontal padding. WYDT?

Reply:
Hmm, maybe removing the padding makes the spacing a little too tight.

<img width="706" alt="image" src="https://github.com/denoland/saaskit/assets/3132889/6499e276-a08b-458f-a5e2-73de6b9e2de7">

I think padding is necessary in order to widen the clickable area of the target.

with no padding:

<img width="380" alt="image" src="https://github.com/denoland/saaskit/assets/3132889/b43197a6-f60c-4b20-a59f-177d38a5cbc7">

with padding:

<img width="412" alt="image" src="https://github.com/denoland/saaskit/assets/3132889/ac41d880-d494-4378-ba13-6640ee15c3cc">